src/components/calcite-dropdown-item/calcite-dropdown-item.scss

```diff
@@ -111,7 +119,8 @@
   }
 }
 
-:host([active]) .container:not(.container--none-selection) {
+:host([active]) .container:not(.container--none-selection),
+:host([active]) .container--link a {
```

Comment:
Can you target the anchor's class (`dropdown-link`) instead of the element? This helps avoid cases where the style may break if the element structure is updated.

This also applies to the selectors above.